python/beeai_framework/adapters/litellm/chat.py

```diff
@@ -275,5 +277,19 @@ def _format_tool_model(self, model: type[BaseModel]) -> dict[str, Any]:
     def _format_response_model(self, model: type[BaseModel] | dict[str, Any]) -> type[BaseModel] | dict[str, Any]:
         return model
 
+    async def clone(self) -> InstanceOf["LiteLLMChatModel"]:
+        kwargs: ChatModelKwargs = {
+            "parameters": ChatModelParameters(**self.parameters.model_dump())
```

Comment:
Why not just `"parameters": self.parameters.model_copy()`?

Reply:
good catch, just an oversight on my part

Reply:
https://github.com/i-am-bee/beeai-framework/pull/705/commits/b2e1541de8e70648d7be72b23d16b7216ee532f0